cEP-0006.md

```diff
@@ -87,3 +83,71 @@ Original text courtesy of the [Speak Up! project] (http://web.archive.org/web/20
+the "This issue is confidential and should only be visible to team members with
+at least Reporter access." checkbox.  We strive to keep all issues confidential.
+
+In your report please include:
+
+* Your contact info (so we can get in touch with you if we need to follow up)
+* Names (real, nicknames, or pseudonyms) of any individuals involved.
+  If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g.
+  a gitter or IRC log) please include a link.
+* Any extra context you believe existed for the incident.
+* If you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+# What happens after you file a report?
+
+You will receive an email from a member coala Code of Conduct Commitee
+acknowledging receipt immediately.  We promise to acknowledge receipt within 24
+hours (and will aim for much quicker than that).
+
+The coala Code of Conduct Committee will immediately meet to review the
+incident and determine:
+
+* What happened.
+* Whether this event constitutes a code of conduct violation.
+* Who the bad actor was.
+* Whether this is an ongoing situation, or if there is a threat to anyone's
+  physical safety.
+* If this is determined to be an ongoing incident or a threat to physical
+  safety, the coala Code of Conduct comittee's immediate priority will be to
+  protect everyone involved.  This means we may delay an "official" response
+  until we believe that the situation has ended and that everyone is physically
+  safe.
+
+Once the coala Code of Conduct Committee has a complete account of the events
+they will make a decision as to how to response.  Responses may include:
+
+* Nothing (if we determine no violation occurred).
+* Giving a warning to the individual(s) involved, either privately or publicly.
+* An imposed vacation (i.e. asking someone to "take a week off" from coala spaces).
+* A permanent or temporary ban from all of coala spaces (gitter, GitHub, GitLab, etc)
+* A request for a public or private apology.
+* We'll respond within one week to the person who filed the report with either
+  a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to
+let them know what action (if any) we'll be taking.   We'll take into account
+feedback from the reporter on the appropriateness of our response, but we don't
+guarantee we'll act on it.
+
+Finally, the coala Code of Conduct Committee will make a report on the situation
+to coala Community Team Leader.
+
+# Conflicts of Interest
+
+In the event of any conflict of interest, or when the incident involves one of
+coala Code of Conduct Committee, the member must immediately notify the other
+members, and recuse themselves if necessary.
```

Comment:
Replacing people has permission implications, such as incoming member having access to old cases about them, and how to prevent the conflicted member from participating (reading) the case potentially involving them, and what to do if there are two cases concurrently, only one with a conflict.

The stand-in is also not going to settle in quickly to meet deadlines mentioned above.

A permanent stand-by non-voting member would reduce these issues.

In my experience it is better to remove the conflicted member immediately, and proceed promptly with a reduced bench, and then reinstate conflicted member or permanently replace them if report warrants it.